src/Geno/C++/GUI/Widgets/TitleBar.cpp

```diff
@@ -63,7 +63,11 @@ void TitleBar::Draw( void )
 
 		m_Height = ImGui::GetWindowHeight();
 
+#if defined( __APPLE__ )
+		ImGui::SetCursorPosX( 67.0f + m_Height * ( 1.0f - IconSizeRatio ) * 0.5f );
```

Comment:
Is there any way to fetch the size of the system button menu, instead of hard-coding a magic number?

Reply:
I realize now that the layout of the system buttons is something we can control. All good 👍